CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
```

Comment:
This should be the same heading level as the versions under it.

Reply:
defmt releases are h2, all others are h3. Fixed it.